docs/OnTrack/Front End Migration/Code References/CoffeeScript/syntax.coffee

```diff
@@ -0,0 +1,186 @@
+# # 1. Variables and Scope
```

Comment:
Those documentation are same as what we did in our team channel. We already asked Nelson to add the Documentation Template to make this documentation make more sense. This is good to me. @ShaeChristmas @Ishrat420 Is there anything that I and Nelson should consider or fix? 
- Moreover, It seems that there is an error relating to linelint in check section of this PR. I have no idea about that error. Let me know if you have any ideas